web/router.ex

```diff
@@ -1,6 +1,10 @@
 defmodule RemoteRetro.Router do
   use RemoteRetro.Web, :router
 
+  if Mix.env == :dev do
+    forward "/sent_emails", Bamboo.EmailPreviewPlug
```

Comment:
Yeah the route name is arbitrary but copied from example, this is basically like the rails letter_opener thing to preview sent mails locally rather than actually sending, similarly dev.exs has config to use LocalAdapter instead of real API, and this plug only works with local adapter.